inc/toolbox.class.php

```diff
@@ -785,16 +785,15 @@ static function stripslashes_deep($value) {
    **/
    static function append_params($array, $separator='&', $parent='') {
 
-      $params = array();
+        $params = array();
       foreach ($array as $k => $v) {
 
          if (is_array($v)) {
             $params[] = self::append_params($v, $separator,
                                             (empty($parent) ? rawurlencode($k)
-                                                            : $parent .'['.rawurlencode($k).']'));
+                                                            : $parent . '%5B' . rawurlencode($k) . '%5D'));
```

Comment:
OK, I see.

The `[]` notation is used on form submission with array of values; if the feature still works as expected; than there is no problem with that change :)